packages/core/src/codecatalyst/explorer.ts

```diff
@@ -44,7 +44,18 @@ async function getLocalCommands(auth: CodeCatalystAuthenticationProvider) {
         iconPath: getIcon('vscode-question'),
     })
 
-    if (!auth.activeConnection || !auth.isConnectionValid()) {
+    // There is a connection, but it is expired, or code catalyst scopes are expired
```

Comment:
use consistent spellings everywhere.
```suggestion
    // There is a connection, but it is expired, or CodeCatalyst scopes are expired.
```